src/script/page/RightSidebar/MessageDetails/MessageDetails.tsx

```diff
@@ -0,0 +1,276 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import type {QualifiedId} from '@wireapp/api-client/src/user/';
+import {WebAppEvents} from '@wireapp/webapp-events';
+import {amplify} from 'amplify';
+import cx from 'classnames';
+import {FC, useCallback, useEffect, useMemo, useState} from 'react';
+
+import {t} from 'Util/LocalizerUtil';
+
+import PanelHeader from '../PanelHeader';
+import Icon from 'Components/Icon';
+import UserSearchableList from 'Components/UserSearchableList';
+import {User} from '../../../entity/User';
+import {ConversationRepository} from '../../../conversation/ConversationRepository';
+import {TeamRepository} from '../../../team/TeamRepository';
+import {Conversation} from '../../../entity/Conversation';
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
+import {SuperType} from '../../../message/SuperType';
+import {UserRepository} from '../../../user/UserRepository';
+import {formatLocale} from 'Util/TimeUtil';
+import {isContentMessage} from '../../../guards/Message';
+import {Message} from '../../../entity/message/Message';
+import {SearchRepository} from '../../../search/SearchRepository';
+import {UserReactionMap} from '../../../storage';
+
+const MESSAGE_STATES = {
+  LIKES: 'likes',
+  NO_LIKES: 'no-likes',
+  NO_RECEIPTS: 'no-receipts',
+  RECEIPTS: 'receipts',
+  RECEIPTS_OFF: 'receipts-off',
+};
+
+const formatUserCount = (users: User[]): string => (users.length ? ` (${users.length})` : '');
+
+const sortUsers = (userA: User, userB: User): number =>
+  userA.name().localeCompare(userB.name(), undefined, {sensitivity: 'base'});
+
+const formatTime = (time: string | number | Date) => formatLocale(time, 'P, p');
+
+interface MessageDetailsProps {
+  activeConversation: Conversation;
+  onClose: () => void;
+  conversationRepository: ConversationRepository;
+  messageEntity: Message;
+  teamRepository: TeamRepository;
+  searchRepository: SearchRepository;
+  userRepository: UserRepository;
+  showLikes?: boolean;
+  updateEntity: (message: Message) => void;
+}
+
+const MessageDetails: FC<MessageDetailsProps> = ({
+  activeConversation,
+  conversationRepository,
+  messageEntity,
+  teamRepository,
+  searchRepository,
+  showLikes = false,
+  userRepository,
+  onClose,
+  updateEntity,
+}) => {
+  const [receiptUsers, setReceiptUsers] = useState<User[]>([]);
+  const [likeUsers, setLikeUsers] = useState<User[]>([]);
+  const [messageId, setMessageId] = useState<string>(messageEntity.id);
+
+  const [isReceiptsOpen, setIsReceiptsOpen] = useState<boolean>(showLikes);
+
+  const {timestamp, user: selfUser} = useKoSubscribableChildren(messageEntity, ['timestamp', 'user']);
```

Comment:
```suggestion
  const {timestamp, user} = useKoSubscribableChildren(messageEntity, ['timestamp', 'user']);
```

This is not the self user (if the name colides with something else, can be named: `messageSender` for example)

Reply:
done.